tests/functional/workflows/rust_cargo/test_rust_actions.py

```diff
@@ -0,0 +1,28 @@
+import os
+import shutil
+import sys
+import tempfile
+import subprocess
+
+from unittest import TestCase
+
+from aws_lambda_builders.workflows.rust_cargo.actions import OSUtils
+
+
+class TestOSUtils(TestCase):
```

Comment:
related to the above comment about copies of osutils. since there is a codecov % barrier to entry, it felt weird having to also copy tests to cover the copied util to keep that codecov %